diff -rupN libgig-4.5.0.old/akai.pc.in libgig-4.5.0/akai.pc.in
--- libgig-4.5.0.old/akai.pc.in	2015-05-02 12:13:28.000000000 +0200
+++ libgig-4.5.0/akai.pc.in	2025-07-17 13:22:16.369889110 +0200
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@/libgig
+libdir=@libdir@
 includedir=@includedir@/libgig
 
 Name: akai
diff -rupN libgig-4.5.0.old/gig.pc.in libgig-4.5.0/gig.pc.in
--- libgig-4.5.0.old/gig.pc.in	2015-05-02 12:13:28.000000000 +0200
+++ libgig-4.5.0/gig.pc.in	2025-07-17 13:22:30.604240400 +0200
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@/libgig
+libdir=@libdir@
 includedir=@includedir@/libgig
 
 Name: gig
diff -rupN libgig-4.5.0.old/src/Makefile.am libgig-4.5.0/src/Makefile.am
--- libgig-4.5.0.old/src/Makefile.am	2025-01-06 12:38:50.000000000 +0100
+++ libgig-4.5.0/src/Makefile.am	2025-07-17 13:23:19.573569684 +0200
@@ -3,6 +3,8 @@ AM_CXXFLAGS = -pedantic -Wreturn-type
 
 pkglib_LTLIBRARIES = libgig.la libakai.la
 
+pkglibdir = @libdir@
+
 libgigincludedir = $(includedir)/libgig
 libgiginclude_HEADERS = RIFF.h DLS.h SF.h gig.h Korg.h Serialization.h sysdef.h
 libgig_la_SOURCES = helper.cpp typeinfo.cpp RIFF.cpp DLS.cpp SF.cpp gig.cpp Korg.cpp Serialization.cpp
