diff -rupN libgig-4.6.0.old/akai.pc.in libgig-4.6.0/akai.pc.in
--- libgig-4.6.0.old/akai.pc.in	2025-11-21 17:25:23.000000000 +0100
+++ libgig-4.6.0/akai.pc.in	2026-08-05 19:30:22.483649350 +0200
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@/libgig
+libdir=@libdir@
 includedir=@includedir@/libgig
 
 Name: akai
diff -rupN libgig-4.6.0.old/gig.pc.in libgig-4.6.0/gig.pc.in
--- libgig-4.6.0.old/gig.pc.in	2025-11-21 17:25:23.000000000 +0100
+++ libgig-4.6.0/gig.pc.in	2026-08-05 19:30:35.733849317 +0200
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@/libgig
+libdir=@libdir@
 includedir=@includedir@/libgig
 
 Name: gig
diff -rupN libgig-4.6.0.old/src/Makefile.am libgig-4.6.0/src/Makefile.am
--- libgig-4.6.0.old/src/Makefile.am	2026-02-07 17:18:12.000000000 +0100
+++ libgig-4.6.0/src/Makefile.am	2026-08-05 19:31:11.954975030 +0200
@@ -3,6 +3,8 @@ AM_CXXFLAGS = -pedantic -Wreturn-type
 
 pkglib_LTLIBRARIES = libgig.la libakai.la
 
+pkglibdir = @libdir@
+
 libgigincludedir = $(includedir)/libgig
 libgiginclude_HEADERS = RIFF.h DLS.h SF.h gig.h Korg.h Serialization.h sysdef.h
 libgig_la_SOURCES = \
